instance/static/js/src/instance.js

```diff
@@ -221,27 +203,8 @@ app.controller("Details", ['$scope', '$state', '$stateParams', 'OpenCraftAPI',
 
             OpenCraftAPI.all("openedx_appserver").post({instance_id: $stateParams.instanceId});
             // The API call above is an asynchronous task so it will return a 200 status immediately.
-            // When the new app server is successfully created, it will send an openedx_appserver_update
-            // notification, which will trigger $scope.refresh(), which will reset 'is_spawning_appserver'
```

Comment:
I tested this.
- With the old behaviour: after clicking the „Launch new AppServer“ button, it becomes disabled for some seconds (and includes a spinner), then becomes enabled again
- With the new behaviour: after clicking the „Launch new AppServer“ button, it becomes disabled for some seconds… and for ever, and nothing else changes. It looks like there was some error (like a JS error), and I feel it won't be very clear to the user that things are fine and we need to wait & refresh. Note that we have a few non-OpenCraft Ocim users

I thought of adding an `alert(…)` to say that it has been correctly spawned and it will be visible after refreshing. We don't use any other `alert`, and it can be intrusive if we use them a lot… (I would still prefer it but I guess others won't). What we have used to show notification is something [like this](https://github.com/open-craft/opencraft/blob/65e2b87a7e3796044b8c90aa928cf081f8312892/instance/static/js/src/openedx_appserver.js#L89), though the notification isn't very prominent (you can actually test this by going to a terminated server, force-enabling the terminate button through your browser, and then clicking the button).
So could you add some message to the user as mentioned above?

Sorry for not foreseeing the eternal spinner in the launch button.

By the way, any other more user-friendly approach can be good too (like: changing the text of the button to something like „launching. Please refresh after some seconds to see the new server“)